packages/text-field/helper-text/README.md

```diff
@@ -65,6 +74,62 @@ class MyApp extends React.Component {
 }
 ```
 
+### Custom Validation
+
+Validation could also be done manually by using the `isValid`, `isValidationMessage` and
+`validation` props, alongside a custom validation method:
+
+```js
+import TextField, {HelperText, Input} from '@material/react-text-field';
+import React from 'react';
+
+class MyApp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {isValid: true, username: ''};
+    this.onChange = this.onChange.bind(this);
+    this.renderHelperText = this.renderHelperText.bind(this);
+  }
+  onChange(e) {
+    const {value} = e.target;
+    this.setState({isValid: value.includes('@'), username: value});
+  }
+  renderHelperText() {
+    if (this.state.isValid) {
+      return (<HelperText>Please enter your Username</HelperText>);
+    } else {
+      return (
+        <HelperText
+          isValid={false}
+          isValidationMessage
```

Comment:
I see what you're saying in #421. let's remove isValidationMessage or validation as well and make them the same. If you wanna make this a different task we can.

Reply:
Let's please solve this through #435 if that's all right.